README.md

```diff
@@ -44,7 +44,27 @@ Add a new value, `$(SRCROOT)/../node_modules/react-native-code-push` and select
 
     ![Add CodePush library reference](https://cloud.githubusercontent.com/assets/516559/10322038/b8157962-6c30-11e5-9264-494d65fd2626.png)
 
-## Plugin Configuration
+## Plugin Installation - Android
+
+To integrate CodePush into your Android project, do the following steps:
+
+1. In your `android/settings.gradle` file, make the following additions:
+    
+    ```gradle
+    include ':app', ':react-native-code-push'
+    project(':react-native-code-push').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-code-push/android/app')
+    ```
+2. In your `android/app/build.gradle` file, add CodePush as one of the dependencies:
```

Comment:
How about...'add the `:react-native-code-push` project as a compile-time dependency:'